Combat/Include/Card.hpp

```diff
@@ -1,8 +1,11 @@
 #pragma once

```

Comment:
Kod który jest tutaj zawarty nie ma żadnego zastosowania. Jest oderwany. To tylko napisane getery i setery, które korzystają z innych geterów i seterów. Bez połączenia tego z czymkolwiek to nie wiadomo, które funkcje są potrzebne, a które nie. Nie czytam całego kodu, bo nie widzę sensu.